src/lib.rs

```diff
@@ -1,71 +1,72 @@
-use std::mem;
-use std::prelude::v1::*;
+#[macro_use]
+extern crate doc_comment;
+doctest!("../README.md");
```

Comment:
I haven't used the `doctest` macro before. Is that the idiomatic location to place the call or should it go into the `test` mod? I'm assuming it's only relevant for testing anyway?

Reply:
I saw it prefixed with `#[cfg(test)]` in the crate root. However, prefixing it with `#[cfg(test)]` and putting it in `dev-dependencies`, I wasn't able to get it to run with `cargo test --doc` :-(. If you have an idea, let me know.